options.json

```diff
@@ -1,6 +1,3 @@
 {
-    "final_workflow_outputs_dir": "results",
-    "default_runtime_attributes": {
-        "docker": "quay.io/lifebitai/ubuntu:18.10"
-    }
-}
\ No newline at end of file
+    "final_workflow_outputs_dir": "results"
```

Comment:
Is this kept for local runs only and will not be used in CloudOS?

Reply:
Yes exactly.

For GEL CloudOS run:
- `options.json`: not used
- `inputs.json`: not used (params are supplied via UI instead)
- Docker settings are supplied in `main.wdl` (see runtime)

For local run:
- `options.json`: used
- `inputs.json`: used - can still run the example command: `java -jar ~/cromwell/cromwell-53.1.jar run main.wdl -i testdata/inputs.json -o options.json`
- Docker settings are supplied in `main.wdl` (see runtime)